infra_tooling_account/infra_tooling_account/infra_tooling_alerting_stack.py

```diff
@@ -22,7 +22,7 @@ class InfraToolingAlertingStack(Stack):
     def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
         self.stage_name = kwargs.pop("stage_name", None)
         self.project_name = kwargs.pop("project_name", None)
-        self.general_settings_reader = kwargs.pop("general_settings_reader", None)
+        self.settings = kwargs.pop("settings", None)
```

Comment:
We have a lot of changes duplicated in our PRs, inevitably.
This is the one of those.

Here, we might want to change  this piece of code to:
self.settings: Settings = kwargs.pop("settings", None)

It helps intellisense a lot, when you start using IDE to call settings methods

Reply:
Didn't know we can do this, honestly 👍 Will rebase on latest changes and do that.